packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -147,10 +147,7 @@ const preparedCharts = computed(() => {
 				height: 120,
 				legend: true,
 				xAxisTitle: modelVarUnits.value._time || 'Time',
-				yAxisTitle:
-					_.uniq(modelVarUnits.value[variable])
-						.filter((v) => !!v)
-						.join(',') || '',
+				yAxisTitle: modelVarUnits.value[variable] || '',
```

Comment:
Why not using a Set?
```js
Array.from(New Set(modelVarUnits.value[variable])).filter((v) => !!v).join(',') || ''
```

Reply:
Optimized chart is a bit different, in that it is always single-variable. I was lazy and copy-pasted the array expression we use elsewhere but that just made things unnecessarily complicated so ...